pkg/proto/http/oidc/oidc.proto

```diff
@@ -21,6 +21,12 @@ message CookieValue {
     // the authorization endpoint URL took place. The user will be
     // redirected back to this page upon success.
     string original_request_uri = 2;
+
+    // The value that was used to compute the code challenge that was
+    // provided to the authorization endpoint URL. It is used to implement
+    // PKCE (RFC 7636) and is sent to the token endpoint URL when
```

Comment:
s/PKCE/Proof Key for Code Exchange/ to improve readability? PKCE as abbreviated looks a bit too much like PKCS (Public Key Cryptography Standards).